### PR TITLE
Fougue.Mayo version 0.6.0

### PR DESCRIPTION
**Pull request by 14AUDDIN**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95480)

### CODE REVIEW

**Comment by 14AUDDIN** [comment]
@microsoft-github-policy-service agree [company="{your company}"]

**Comment by 14AUDDIN** [comment]
@microsoft-github-policy-service agree
@microsoft-github-policy-service agree company="your company"

**Comment by 14AUDDIN** [comment]
@microsoft-github-policy-service agree

**Comment by 14AUDDIN** [comment]
@microsoft-github-policy-service agree

File: manifests/f/Fougue/Mayo/0.6.0/Fougue.Mayo.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Fougue.Mayo
+PackageVersion: 0.6.0
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/fougue/mayo/releases/download/v0.6.0/mayo_v0.6.0.1012-e55c5ee_winx64_installer.exe
+  InstallerSha256: 12BC7DF1973C58BA31DB4F0E7FE751088B04BEB5016731EB54BBF9333A5D7A6A
```
Comment (mdanish-kh) [inline manifests/f/Fougue/Mayo/0.6.0/Fougue.Mayo.installer.yaml:10]:
```suggestion
  InstallerSha256: 12BC7DF1973C58BA31DB4F0E7FE751088B04BEB5016731EB54BBF9333A5D7A6A
  AppsAndFeaturesEntries:
    - DisplayVersion:
```
The installer doesn't write a DisplayVersion to Registry/Control Panel, need to explicitly state this in the manifest.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**